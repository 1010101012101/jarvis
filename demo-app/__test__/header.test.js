
import { h } from 'preact';
import render from 'preact-render-to-string';
import Header from '../components/header';


describe('Header component', () => {
	it('should render header with content', () => {
		const tree = render(
			<Header content="Zouhir" />
		);
		expect(tree).toMatchSnapshot();
  });
  it('should render header with default fallback', () => {
		const tree = render(
			<Header />
		);
		expect(tree).toMatchSnapshot();
	});
});