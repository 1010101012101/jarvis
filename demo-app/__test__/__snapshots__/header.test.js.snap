// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render header with content 1`] = `
"<h1 class=\\"
  color: #f00;
\\">Zouhir</h1>"
`;

exports[`Header component should render header with default fallback 1`] = `
"<h1 class=\\"
  color: #f00;
\\">unprovided</h1>"
`;
